Project.toml

```diff
@@ -6,11 +6,13 @@ version = "0.2.0"
 [deps]
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
please do not introduce unneeded dependencies. Revise and Run are both very useful packages but not inseparable from this package. Users can have them if they wish already installed in the general environment or whatever.